crates/electrum/src/electrum_ext.rs

```diff
@@ -165,20 +165,15 @@ pub trait ElectrumExt<A> {
     /// The scan for each keychain stops after a gap of `stop_gap` script pubkeys with no associated
     /// transactions. `batch_size` specifies the max number of script pubkeys to request for in a
     /// single batch request.
```

Comment:
You should also improve the documentation explaining when to use this method, as described in #1112